### DISCUSSION TITLE
Long query string arrays converted to objects (express issue)

### DISCUSSION THREAD

**Discussion by christo911** [loopbackio/loopback-next#7581] [Q&A] [answered]
When using get request with filter including an array of more than strings, the filter is converted from an array to an object.
This is related to express qs, as described exactly here: 

[https://github.com/expressjs/body-parser/issues/289](url)

It also seems this was an issue in the past with loopback 3, and was resolved by increasing the querysting Array limit, as described here:

[https://github.com/strongloop/loopback/issues/2824](url)

However, how can this be set in Loopback 4?

```
const app = new BackendLb4Application(options)
app.set // not available
```

Also the options, only allows for the settings of the requestBodyParser, not the express querystring (qs), which it is using.

**Comment by marioestradarosa** [comment, marked as answer]
Have you tried to stringify your filter before sending it to loopback 4 ?

**Reply by christo911** [reply]
Ahhhh brilliant! thanks mariosestradorsa!
Well solved!
